drivers/network/dd/e1000/hardware.c

```diff
@@ -53,26 +53,25 @@ static USHORT SupportedDevices[] =
     0x10B5,     // Intel 82546GB Quad Copper KSP3
 };
 
-
-static ULONG E1000WriteFlush(IN PE1000_ADAPTER Adapter)
+static
+inline
```

Comment:
`static` and `static inline` have no practical difference, so I'd like to leave only `static`

Reply:
Why? Compiler behaves in different ways for `static` and `static inline`